### ISSUE TITLE
testkit cannot be resolved

### ISSUE THREAD

**Issue by olafurpg** [scalacenter/scalafix#612]
The scalafix.g8 template CI failed because testkit cannot be resolved anymore due to its dependency on test-utils

```
$ coursier fetch ch.epfl.scala:scalafix-testkit_2.12.4:0.6.0-M1                                                                                                      
Error:
  ch.epfl.scala:scalafix-test-utils_2.12:0.6.0-M1
    not found: /Users/ollie/.ivy2/local/ch.epfl.scala/scalafix-test-utils_2.12/0.6.0-M1/ivys/ivy.xml
    not found: https://repo1.maven.org/maven2/ch/epfl/scala/scalafix-test-utils_2.12/0.6.0-M1/scalafix-test-utils_2.12-0.6.0-M1.pom
```

https://travis-ci.org/scalacenter/scalafix.g8/builds/341912036?utm_source=email&utm_medium=notification

Can we either publish test-utils or drop the dependency on test-utils in testkit? Alternatively, can we inline test-utils into testkit if we want to make it part of the public API?

**Comment by olafurpg** [comment]
I've temporarily hardcoded the scalafix.g8 template to use scalafix 0.5.10

**Comment by olafurpg** [comment]
Fixed in #740
